websites/serializers.py

```diff
@@ -233,12 +233,21 @@ def get_url_suggestion(self, instance):
         return instance.get_url_path(with_prefix=False)
 
 
+class WebsiteHasMetadataMixin(serializers.Serializer):
+    has_site_metadata = serializers.SerializerMethodField(read_only=True)
```

Comment:
It's always a good idea to add [doc strings](https://www.geeksforgeeks.org/python-docstrings/). 

```suggestion
class WebsiteHasMetadataMixin(serializers.Serializer):
    """Adds has_site_metadata custom serializer field."""
    has_site_metadata = serializers.SerializerMethodField(read_only=True)
```